### DISCUSSION TITLE
health checks

### DISCUSSION THREAD

**Discussion by AntonyKing7D** [drakkan/sftpgo#665] [Ideas]
Hi,

Looking at the code, does this imply that /healthz will always return 'ok' so long as the application is running?
```
        s.router.Get(healthzPath, func(w http.ResponseWriter, r *http.Request) {
                render.PlainText(w, r, "ok")
        })

```
Would it be plausible to check the database and backend connections as part of this check ?

**Comment by drakkan** [comment]
Hi,

to get a more complete server status there is a different API:

https://sftpgo.stoplight.io/docs/sftpgo/b3A6Mjc0MjY3NA-get-status

and to get active connections there is another API:

https://sftpgo.stoplight.io/docs/sftpgo/b3A6NzUwODM4MA-get-connections-details

Database availability, client connections and other details are also available as prometheus metrics if you enable the telemetry server

**Reply by AntonyKing7D** [reply]
Thanks, that's really helpful.
